src/README.md.jinja

```diff
@@ -12,7 +12,12 @@
     {{- version | replace(".","-") -}}
 {%- endmacro -%}
 [![Runbot Status](https://runbot.odoo-community.org/runbot/badge/flat/{{ repo_id(repo_slug) }}/{{ odoo_version }}.svg)](https://runbot.odoo-community.org/runbot/repo/github-com-oca-{{ repo_slug }}-{{ repo_id(repo_slug) }})
+{% if ci == "Travis" %}
 [![Build Status](https://travis-ci.com/OCA/{{ repo_slug }}.svg?branch={{ odoo_version }})](https://travis-ci.com/OCA/{{ repo_slug }})
+{% else %}
+[![Pre-commit Status](https://github.com/OCA/{{ repo_slug }}/actions/workflows/pre-commit.yml/badge.svg)](https://github.com/OCA/{{ repo_slug }}/actions/workflows/pre-commit.yml)
+[![Build Status](https://github.com/OCA/{{ repo_slug }}/actions/workflows/test.yml/badge.svg)](https://github.com/OCA/{{ repo_slug }}/actions/workflows/test.yml)
```

Comment:
This badge is for both OCB and Odoo?

Reply:
Yes, OCB and Odoo are tested in the same GitHub workflow.